variables.mk

```diff
@@ -1,23 +1,45 @@
 #########################################################################################
 # makefile variables shared across multiple makefiles
-#########################################################################################
-
-#########################################################################################
-# variables to invoke the generator
-# descriptions:
-#   SBT_PROJECT = the SBT project that you should find the classes/packages in
-#   MODEL = the top level module of the project in Chisel (normally the harness)
-#   VLOG_MODEL = the top level module of the project in Firrtl/Verilog (normally the harness)
-#   MODEL_PACKAGE = the scala package to find the MODEL in
-#   CONFIG = the configuration class to give the parameters for the project
-#   CONFIG_PACKAGE = the scala package to find the CONFIG class
-#   GENERATOR_PACKAGE = the scala package to find the Generator class in
-#   TB = wrapper over the TestHarness needed to simulate in a verilog simulator
-#   TOP = top level module of the project (normally the module instantiated by the harness)
-#
-# project specific:
-# 	SUB_PROJECT = use the specific subproject default variables
-#########################################################################################
+# - to use the help text, your Makefile should have a 'help' target that just 
```

Comment:
Nit: I think 99.999999% of people will know about `make help` by default, but it might also be useful to mention this in the documentation. However, that is a non-blocking issue.

Reply:
updated the docs.